editor/project_settings_editor.cpp

```diff
@@ -370,17 +370,7 @@ void ProjectSettingsEditor::_action_edited(const String &p_name, const Dictionar
 
 	} else {
 		// Events changed
-		int act_event_count = ((Array)p_action["events"]).size();
-		int old_event_count = ((Array)old_val["events"]).size();
-
-		if (act_event_count == old_event_count) {
```

Comment:
Why remove that ? More precise messages are welcome.

Reply:
Because I don't think it is an accurate reflection of what happneded 100% of the time now. Say you have 5 events which are all different to the defaults. Then you reset to default. What happened technically is you just removed 3 events and changed 2 events. But with this message it would just say 'Remove Input Action Event' - which is not accurate as more than one was removed and 2 others were changed.

Eaiser to just make it a generic message imo